packages/procaptcha-react/src/components/CaptchaWidget.tsx

```diff
@@ -91,58 +83,70 @@ export const CaptchaWidget = ({ challenge, solution, onClick }: CaptchaWidgetPro
                                     src={item.data}
                                     alt={`Captcha image ${index + 1}`}
                                 />
-                            </Box>
-                            <Fade in={solution.includes(hash)}>
-                                <Box
-                                    sx={{
-                                        // relative to where the element _should_ be positioned
-                                        position: 'relative',
-                                        // make the overlay the full height/width of an item
-                                        width: '100%',
+                            </div>
+
+                            <div
+                                style={{
+                                    // relative to where the element _should_ be positioned
+                                    position: 'relative',
+                                    // make the overlay the full height/width of an item
+                                    width: '100%',
+                                    height: '100%',
+                                    // shift it up 100% to overlay the item element
+                                    top: '-100%',
+                                    // transition on opacity upon (de)selection
+                                    //transitionDuration: '300ms',
```

Comment:
Can we remove these unused commented out properties?

In general there's a few comments in this bit that could easily become stale and confusing

Reply:
:+1: